From 3443f2a4e0563fda7f11199f8446ca29efc0d3ac Mon Sep 17 00:00:00 2001
From: Max Lee <yowching.lee@gmail.com>
Date: Mon, 30 Mar 2020 00:26:17 +0800
Subject: [PATCH] Remove unused variable...

---
 json_tokener.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/json_tokener.c b/json_tokener.c
index 19de8ef..9a76293 100644
--- a/json_tokener.c
+++ b/json_tokener.c
@@ -352,12 +352,10 @@ struct json_object* json_tokener_parse_ex(struct json_tokener *tok,

     case json_tokener_state_inf: /* aka starts with 'i' */
       {
-	int size;
 	int size_inf;
 	int is_negative = 0;
 
 	printbuf_memappend_fast(tok->pb, &c, 1);
-	size = json_min(tok->st_pos+1, json_null_str_len);
 	size_inf = json_min(tok->st_pos+1, json_inf_str_len);
 	char *infbuf = tok->pb->buf;
 	if (*infbuf == '-')
--
2.25.1

