From 8f71a87a9faf075fb3253dd240ffd06e4f891052 Mon Sep 17 00:00:00 2001
From: Max Lee <yowching.lee@gmail.com>
Date: Fri, 1 May 2020 02:08:53 +0800
Subject: [PATCH 2/3] Extent maximum prompt size

---
 inc/cparser_options.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/inc/cparser_options.h b/inc/cparser_options.h
index 289d9f8..82a4c64 100644
--- a/inc/cparser_options.h
+++ b/inc/cparser_options.h
@@ -37,7 +37,7 @@
  * Maximum number of character of a command prompt including the 
  * terminating NULL.
  */
-#define CPARSER_MAX_PROMPT         (16)
+#define CPARSER_MAX_PROMPT         (1024)
 
 /**
  * Maximum number of nested sub-mode levels.
-- 
2.26.2

