From fe1a76c3a1b5ece7cd1f27ec24129973804895e7 Mon Sep 17 00:00:00 2001
From: Max Lee <yowching.lee@gmail.com>
Date: Fri, 1 May 2020 02:09:37 +0800
Subject: [PATCH 3/3] Make compiler happy

---
 src/test_cli_cmd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/test_cli_cmd.c b/src/test_cli_cmd.c
index 17b069f..5b0232f 100644
--- a/src/test_cli_cmd.c
+++ b/src/test_cli_cmd.c
@@ -285,7 +285,7 @@ cparser_cmd_emp_name_name (cparser_context_t *context, char **name)
     assert(context && name && (*name));
     emp = (employee_t *)context->cookie[1];
     assert(emp);
-    strncpy(emp->name, *name, MAX_NAME);
+    strncpy(emp->name, *name, MAX_NAME-1);
     return CPARSER_OK;
 }
 
-- 
2.26.2

